### ISSUE TITLE
Add `-sex` flag to `sct_compute_compression`

### ISSUE THREAD

**Issue by valosekj** [sct-pipeline/dcm-metric-normalization#14]
Add [`-sex` flag](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d9f2beddc269e45835ace94db23eee4e65308d9b/spinalcordtoolbox/scripts/sct_compute_compression.py#L121-L126) to `sct_compute_compression` to normalize metrics per sex during processing with [process_data_dcm-zurich.sh](https://github.com/sct-pipeline/dcm-metric-normalization/blob/657966502487f56cd61147af7ece740a91d8ad8b/scripts/process_data_dcm-zurich.sh#L183-L201) script.

Basically, we just need to fetch sex from `participants.tsv` file, for example using:

```
grep ${SUBJECT} participants.tsv | awk '{print $5}'
```
